sentry_sdk/scope.py

```diff
@@ -931,30 +1140,27 @@ def _capture_internal_exception(
     def start_session(self, *args, **kwargs):
         # type: (*Any, **Any) -> None
         """Starts a new session."""
-        client = kwargs.pop("client", None)
         session_mode = kwargs.pop("session_mode", "application")
 
-        self.end_session(client=client)
+        self.end_session()
 
+        client = Scope.get_client()
         self._session = Session(
-            release=client.options["release"] if client else None,
-            environment=client.options["environment"] if client else None,
+            release=client.options["release"],
```

Comment:
Are these defined on `NoOpClient`s?

Reply:
Yes. All the parameters defined [here](https://github.com/getsentry/sentry-python/blob/new-scopes-call-api/sentry_sdk/consts.py#L252) are used as default options in all clients. But changed them to `.get()` to be extra safe.